scene/main/node.cpp

```diff
@@ -757,7 +757,7 @@ void Node::rpc_config(const StringName &p_method, const Variant &p_config) {
 	}
 }
 
-const Variant Node::get_node_rpc_config() const {
+Variant Node::get_node_rpc_config() const {
```

Comment:
The name of this method is a bit weird, which it didn't matter before as it's not exposed. Saying "node" is like it's asking for a node parameter to be passed, but the subject of interest is the node calling this method. I'm aware that `is_node_ready` exists too, but we [don't seem particularly fond of that name](https://github.com/godotengine/godot-proposals/issues/6858) either.
Furthermore, these two methods essentially do the same thing and do not conflict with one another(?). I may be wrong on this, which justifies the naming scheme.

Reply:
I had to dig into it a bit to determine the difference between the two functions. The script's `get_rpc_config()` reflects the config set with the RPC annotation, and `get_node_rpc_config()` are exclusively the values set at runtime with `rpc_config()` (leading to some internal differences in SceneRPCInterface, I think?).

Maybe something like `get_dynamic_rpc_config` would be more fitting for Node (note SceneRPCInterface caches the config state as well, so it's technically not entirely dynamic either, but it helps assert the distinction between the two)